src/mapping.ts

```diff
@@ -488,4 +490,18 @@ export function handleAssessorFile(call: AssessorV3FileCall): void {
   }
 
   pool.save()
+}
+
+export function handleERC20Transfer(event: TransferEvent): void {
+  createToken(event)
+  loadOrCreateTokenBalanceDst(event)
+  loadOrCreateTokenBalanceSrc(event)
+  updateAccounts(event)
+
+  let poolMeta = poolFromAddress(event.address)
+  let id = event.block.number.toString().concat('-').concat(event.logIndex.toString())
+  let transfer = ERC20Transfer.load(id)
+  if(transfer == null) {
+    transfer = createERC20Transfer(id, event, poolMeta)
+  }
```

Comment:
Since you're not returning the `transfer` key, I think this can be simplified:

```suggestion
  if (ERC20Transfer.load(id) == null) {
    createERC20Transfer(id, event, poolMeta)
  }
```